.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Please add .idea to your global gitignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c